design/one-pager-definition-of-done.md

```diff
@@ -0,0 +1,40 @@
+# Definition of done
+
+* Owner: Daniel Suskin (@suskin)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Introduction
+
+*Why does this document exist?*
+
+The goal of this document is to answer the questions:
+
+* As a developer, when should I consider a chunk of work to be complete?
+* As a reviewer, what should I be looking for to verify that a chunk of work is
+  complete?
+
+## Definition of work
+
+Work means a contribution to the project, typically code.
+
+## Definition of done
+
+A chunk of work is considered done when the following criteria have been met:
+
+* All code is written
+* Code is reviewed and approved
+* New behavior is documented in the documentation that will go onto the
+  crossplane website
+* All error conditions are reported into the log or as an event, as
+  appropriate
+* Code is tested, ideally in an integration test
+
+## Applying definition of done
+
+The definition of done can be applied in the following ways:
+
+* Pull requests should not be merged until the work contained in them is
+  done.
```

Comment:
Is there a way we could integrate this document with the existing checklist that is exposed as part of the [Github PR template](https://github.com/crossplaneio/crossplane/blob/master/.github/PULL_REQUEST_TEMPLATE.md)?

Reply:
Ooo, nice! Let me take a closer look at this.

Reply:
I have done some integration now